crawler/cmd/one.go

```diff
@@ -59,25 +65,22 @@ func getPAfromWhiteList(repoURL string, args []string) (pa crawler.PA) {
 		// looking into repositories
 		for _, paWlRepo := range paWl.Repositories {
 			log.Tracef("matching %s with %s", paWlRepo, repoURL)
-			if paWlRepo == repoURL {
+			if (url.URL)(paWlRepo) == repoURL {
 				log.Debugf("PA found in whitelist %+v", paWl)
 				return paWl
 			}
 		}
 		// looking into organizations
 		for _, paWlRepo := range paWl.Organizations {
 			log.Tracef("matching %s.* with %s", paWlRepo, repoURL)
-			if matched, _ := regexp.MatchString(paWlRepo+".*", repoURL); matched {
-				log.Debugf("PA found in whitelist %+v", paWl)
+			if matched, _ := regexp.MatchString(paWlRepo.String()+".*", repoURL.String()); matched {
+				log.Debugf("Publisher found in whitelist %+v", paWl)
 				return paWl
 			}
 		}
 	}
 
-	log.Warn("PA not found in whitelist, slug will be generated without coideIPA")
-	// since this routine is called by command: `<command_name> one ...`
-	// that is not aware about whitelists
-	// this hack will skip IPA code match with those lists
-	pa.UnknownIPA = true
-	return pa
+	log.Warn("Publisher not found in whitelist, slug will be generated without Id")
```

Comment:
**[suggestion]** We could log also the name/id of the publisher not found in the whitelist?

Reply:
Good point, opening it as a new issue.